kmath-tensors/src/commonTest/kotlin/space/kscience/kmath/tensors/core/TestBroadcasting.kt

```diff
@@ -0,0 +1,104 @@
+package space.kscience.kmath.tensors.core
+
+import space.kscience.kmath.tensors.core.algebras.DoubleTensorAlgebra
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class TestBroadcasting {
```

Comment:
Test classes are usually made `internal` to prevent problems if someone imports a test source set.

Reply:
Not necessary, but won't harm.